src/anemoi/models/models/encoder_processor_decoder.py

```diff
@@ -69,6 +70,31 @@ def __init__(
 
         self.num_channels = config.model.num_channels
 
+        # If self.layer_kernels entry is missing from the config, use torch.nn by default
+        try:
+            self.layer_kernels = config.model.layer_kernels
+        except AttributeError:
+            LOGGER.info("No entry found for config.model.layer_kernels. Defaulting to torch.nn implementations")
+            self.layer_kernels = DotDict()
+            self.layer_kernels["Linear"] = DotDict({"_target_": "torch.nn.Linear", "_partial_": True})
+            self.layer_kernels["LayerNorm"] = DotDict({"_target_": "torch.nn.LayerNorm", "_partial_": True})
+
+        # try loading each of the requested kernels
+        # If a given kernel isnt availible, fallback to the torch.NN implementation of the same name
+        # TODO I would prefer to come up with a way for hydra to loop over options to instiate, rather then having to catch errors like this
```

Comment:
spell check needed; also, is the TODO out of scope for this PR?

Reply:
sure, It was more of a note to check if Hydra could do this. But I'll remove it